elyra/pipeline/processor_kfp.py

```diff
@@ -169,6 +79,126 @@ def cc_pipeline():
 
         return None
 
+    def export(self, pipeline):
+        if pipeline.file_type not in ["tgz", "tar.gz", "zip", "yaml", "yml", "py"]:
+            self.log.warn("Pipeline file type not recognized...defaulting to tar.gz")
+            pipeline_file_ext = "tar.gz"
+        else:
+            pipeline_file_ext = pipeline.file_type
```

Comment:
Also, I'm torn between "file type" and "extension".  These are extensions, but they drive the type of file produced.  Just wondering if we should use consistent naming and change to `pipeline_file_type`.

Reply:
updated to pipeline_file_type to maintain consistency